lec12-datasets.Rmd

```diff
@@ -0,0 +1,189 @@
+    - (Try to) mediate or resolve any conflicts (there will *always* be some type of conflict; how it's dealt with is what matters)
+    - (Try to) encourage everyone to participate and allow everyone a chance at talking
+- Recorder's duty:
+    - Goal is to write/type down main or important points raised or discussed when team is meeting
+    - Keep notes and files organized and orderly
+- Organizer's duty:
+    - Arrange for next meeting time and location
+    - Send reminders to members a day or two before meeting
+
+### Code of conduct
+
+- Vital to establishing boundaries and expectations of being a team member
+    - How do you want each member to treat each other?
+    - How do you deal with conflict?
+    - What is acceptable and unacceptable behaviour?
+- These are outlined in the code of conduct
+- Mostly its common sense (be kind and respectful)
+    - But its important that you as a team write out what everyone wants and agrees to
+
+## Exercise
+
+### As a group, complete these tasks
+
+<!-- show assigned groups -->
+
+- Get into your assigned groups
+- Introduce each other:
+    - Find out everyone's name and year of study
+    - Find out other things about each other (e.g. any plans for next year, etc)
+- Create a two or one word "team name". We'll use this to create a shared folder for everyone to use.
+- Assign roles to each person (these roles will be rotated in every group setting):
+    - You need: facilitator, recorder, organizer
+    - Discuss how and when roles will be rotated *(record it)*
+- Discuss and brainstorm some codes of conduct you want your team to follow *(record it)*
+- Take a few minutes, think about your own skills and what you feel are most competent in
+    - Then, share the top one or two of those skills *(record those skills)*
+- Discuss how responsibilities of each member will be decided on *(record it)*
+
+## Starting the projects
+
+### Datasets available for use
+
+- [Continuous Plankton Recorder Dataset](https://www.gbif.org/dataset/67c54f85-7910-4cbf-8de4-6f0b136a0e34)
+
+    Data on northern hemisphere plankton species, latitude, longitude, date.
+    Going back to 1946. 935 Mb size, almost 2.75 million rows of data.
+
+- [Insecta of Costa Rica](https://www.gbif.org/dataset/3e9817c1-8302-4955-87e3-a408db0ea379)
+
+    Data on insects species in Costa Rica, latitude, longitude, elevation, date.
+    1.4 Gb size, almost 3.25 million rows of data.
+
+- [Marine predator and prey body sizes](http://www.esapubs.org/archive/ecol/E089/051/default.htm#data)
+
+    Data from 27 different global locations on species, body measurements,
+    latitude, longitude, date. 21 Mb, almost 35,000 rows of data (in long
+    format).
+
+- [Mammalian life history](http://www.esapubs.org/archive/ecol/E084/093/default.htm)
+
+    Data about general mammalian life history with species, body size, lifespan,
+    litter size, and other reproductive variables. 150 Kb size, 1440 rows of
```

Comment:
Seems miniscule in comparison...

Reply:
Yes, I know. But the benefit of this dataset is the richness in the variables, so you can ask much more diverse questions.

Reply:
And the type of analyses could be much more diverse as well. However, you don't have the analytic challenge of the massive size of the other datasets... That's what I was going to mention anyway.